src/main/java/com/acmcsuf/bot_committee/Trigger.java

```diff
@@ -55,6 +55,12 @@ public void onSlashCommandInteraction( @NotNull SlashCommandInteractionEvent eve
 
         if ( !authenticateMemberByRole( event.getMember() ) )
         {
+            EmbedBuilder builder = new EmbedBuilder()
+                    .setColor( Color.red )
+                    .setDescription( "You are not authorized to use this feature!" )
+                    .setDescription( "If you believe this is a mistake, contact your server admin" );
```

Comment:
Style nit: Member-facing text should be complete sentences.
```suggestion
                    .setDescription( "If you believe this is a mistake, contact your server admin." );
```